### DISCUSSION TITLE
Using double-quotes to denote inches [GUI/helptext]

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1089] [Q&A] [answered]
It is incorrect to represent "three and a half inch" with "3.5' '", in the same way it's incorrect to represent "three and a half centimeters" with "3.5c m"

I do realize some of the characters discussed here, have secret superhero lives as special operators in code...m'kay? I'm more focused on what looks visually correct, and what is proper usage conventions (when it comes to the case of not all keymaps including single prime)

If I let my childhood imperial measurement mind read 3.5' ' it literally states "three and a half feet ...with trailing apostrophe/single quote"... (and the second bad example might be 3.5 degrees C per meter =)

I'm reading through PanelFloppy.cpp helptext area, but know there's other places 3.5' ' exist.

FTR the grammatic rules are like this: when a single/double quote (nee prime) is being used to denote a measurement, that functionality takes precedent over the single/double quote's other uses in grammar, and are considered bound to the descriptor (the number, 3.5). When you need to quote a portion of text that  contains the single/double quotes being used in this fashion, the quotation is encapsulated by single quotes (in lieu of single prime)...ergo..

With the buttons "Create 3.5'' DD disk" and "Create 3.5'' HD disk" you can

...should be...

With the buttons 'Create 3.5" DD disk' and 'Create 3.5" HD disk' you can

TBH I had to go look that up...just because I lived through the time AU converted to metric, and the context of expressing feet/inches as '/" has long been deprecated...and this is obscure grammar 8)

My question here is, why don't we use double quotes to denote inches, when such is allowed substitution and visually more appealing/correct?

TIA

**Comment by midwan** [comment, marked as answer]
This is absolutely a bug, which was missed. I'll get it fixed, thanks for noticing it.

**Comment by giantclambake** [comment]
Excellent, thank you ~ it's been really annoying me 8)
